src/bridge/SequencerInbox.sol

```diff
@@ -437,8 +437,38 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         IGasRefunder gasRefunder,
         uint256 prevMessageCount,
         uint256 newMessageCount
+    ) external refundsGas(gasRefunder, reader4844) {
+        revert Deprecated();
+    }
+
+    function addSequencerL2BatchFromBlobs(
+        uint256 sequenceNumber,
+        uint256 afterDelayedMessagesRead,
+        IGasRefunder gasRefunder,
+        uint256 prevMessageCount,
+        uint256 newMessageCount,
+        bytes memory quote
     ) external refundsGas(gasRefunder, reader4844) {
         if (!isBatchPoster[msg.sender]) revert NotBatchPoster();
+
+        bytes32[] memory dataHashes = reader4844.getDataHashes();
+        if (dataHashes.length == 0) revert MissingDataHashes();
+        // take keccak2256 hash of all the function arguments and encode packed blob hashes
+        // except the quote
+        bytes32 reportDataHash = keccak256(
+            abi.encode(
+                sequenceNumber,
```

Comment:
I see that the order of the elements encoding is not similar to other `addSequencer` functions, why this specific order?

Reply:
Its based on the function arguments and their ordering